.coveragerc

```diff
@@ -1,4 +1,5 @@
 [run]
+concurrency = multiprocessing
```

Comment:
Codecov reports only ~12% patch coverage for ``flfm/cli.py`` because that code is executed on a separate process using ``multiprocess``. Adding the above setting works and local codecov tests show expected patch coverage. However, this is not the case for the CI. The [archived ``coverage.xml`` file](https://github.com/ssec-jhu/flfm/actions/runs/16482843683/job/46601261173#step:7:31) shows no patch coverage for ``flfm/cli.py``.